### ISSUE TITLE
release 0.15.0?

### ISSUE THREAD

**Issue by dcherian** [pydata/xarray#3702]
It's been 2 months since our last release. It'd be nice to issue one soon; mostly to get the really nice documentation changes out.

According to our min versions policy we can bump up to numpy 1.15 and dask 2.2 (though we need to solve #3660 first). So I propose we do that and bump to `0.15.0`

#### must-have:
- [x] #3660 
- [x] #3713
- [x] #3560 
- [x] #3673 - waiting on upstream fix.

#### close/ready and nice to have:
- [ ] #3691
- [ ] #3588
- [x] #3670 
- [x] #3631
- [ ] #3611 
- [ ] #3610 
- [x] #3618

**Comment by keewis** [comment]
I'd say all of the PRs related to #3594 (making all non-indexing tests pass and documentation) should be put into the same release. There is not too much to do so if I get about a week I can make sure all of them are ready for review / merge, but I'm also fine with not including them in 0.15

**Comment by dcherian** [comment]
Should be ready to go once #3631 is in.

**Comment by dcherian** [comment]
Do we have volunteers to do this release?

**Comment by dcherian** [comment]
I am doing this now unless there are objections.

**Comment by keewis** [comment]
no objections from me

**Comment by dcherian** [comment]
```
git clean -xdf; python setup.py bdist_wheel sdist; twine check dist/*
```

gives me 
```
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 39: Warning: Bullet list ends without a blank line; unexpected unindent.
```

Can someone figure what needs to be fixed  please?

**Comment by rabernat** [comment]
Thanks @dcherian for handling this release!

I believe this is the problem. (line 39 of the description is like 47 of the file)

https://github.com/pydata/xarray/blob/7c32c8ae77557178d618abe65924baf89667b55c/setup.cfg#L46-L47

I guess we are not continuing the line properly.

**Comment by keewis** [comment]
the reason for this is that we have bullet lists with more than one line per item. `python setup.py sdist` (and others) will reindent the lines not preceded by the bullet (`-`) to match the indent of the bullet.

Is there a reason why we put a string there instead of using something like `file: README.rst`?

**Comment by dcherian** [comment]
> will reindent the lines not preceded by the bullet (-) to match the indent of the bullet.

That was it!

> Is there a reason why we put a string there instead of using something like file: README.rst?

No idea. let's tackle this after this release.

**Comment by keewis** [comment]
did you tag already? If not, I think it might be good to get #3732 in.

**Comment by dcherian** [comment]
OK everything but the email, twitter announcements is done. I'll get to that tonight/tomorrow

**Comment by dcherian** [comment]
great RTD is now failing consistently: https://readthedocs.org/projects/xray/builds/

**Comment by keewis** [comment]
that's because `conda` uses too much memory. We might be able to reduce the memory consumption by moving most packages into the `pip` section (except from `iris` and `cartopy` every package should have wheels PyPI)

**Comment by max-sixty** [comment]
Thanks @dcherian !

And @keewis for being so helpful recently!

**Comment by dcherian** [comment]
Update: RTD is fixed but the conda-forge channel needs some attention: https://github.com/conda-forge/xarray-feedstock/pull/54

**Comment by crusaderky** [comment]
```
    ERROR: Could not find a version that satisfies the requirement setuptools_scm (from versions: none)
    ERROR: No matching distribution found for setuptools_scm
```

Should get fixed by changing recipe/meta.yaml:
```yaml
requirements:
  build:
    - python >=3.6
    - pip
    - setuptools
    - setuptools_scm
```

**Comment by dcherian** [comment]
Done!
